Makefile

```diff
@@ -149,8 +198,11 @@ CPPFLAGS += -P -xc -fno-dollars-in-identifiers
 
 # Converts e.g. ntsc-1.0 to NTSC_1_0
 VERSION_MACRO := $(shell echo $(VERSION) | tr a-z-. A-Z__)
-CPP_DEFINES += -DOOT_VERSION=$(VERSION_MACRO)
+CPP_DEFINES += -DOOT_VERSION=$(VERSION_MACRO) -DOOT_REVISION=$(REVISION)
 CPP_DEFINES += -DOOT_REGION=REGION_$(REGION)
+CPP_DEFINES += -DBUILD_CREATOR="\"$(BUILD_CREATOR)\"" -DBUILD_DATE="\"$(BUILD_DATE)\"" -DBUILD_TIME="\"$(BUILD_TIME)\""
```

Comment:
nit: the unescaped quotes here don't do anything, bash would interpret it the same if it were `-DBUILD_CREATOR=\"$(BUILD_CREATOR)\"`. I think `-DBUILD_CREATOR='"$(BUILD_CREATOR)"'` or `'-DBUILD_CREATOR="$(BUILD_CREATOR)"'` would work too if you think that's cleaner

Reply:
If I remove the outer quotes would it still work if `BUILD_CREATOR` had spaces in it?

Reply:
hmm good point, not sure. Never mind then